lib/smart_proxy_remote_execution_ssh/actions/pull_script.rb

```diff
@@ -81,6 +80,21 @@ def mqtt_start(otp_password)
       output[:state] = :notified
     end
 
+    def mqtt_cancel
+      cleanup
+      payload = {
+        type: 'data',
+        message_id: SecureRandom.uuid,
+        version: 1,
+        sent: DateTime.now.iso8601,
+        directive: 'cancel',
```

Comment:
The directive needs to be `foreman`, that informs the yggdrassil what worker to deliver this to.
I guess we will have to put the cacel in the metadata in some way